apps/andi/lib/andi_web/input_schemas/ingestion_metatadata_form_schema.ex

```diff
@@ -26,14 +26,39 @@ defmodule AndiWeb.InputSchemas.IngestionMetadataFormSchema do
     :targetDataset
   ]
 
+  def extract_from_ingestion_changeset(%Ecto.Changeset{ data: %Andi.InputSchemas.Ingestion{} } = ingestion_changeset) do
+    ingestion_data = ingestion_changeset
```

Comment:
apply_changes gets the changed data regardless of any validations, aka, the complete latest data